java/src/main/java/org/vosk/Recognizer.java

```diff
@@ -0,0 +1,34 @@
+package org.vosk;
+
+import com.sun.jna.PointerType;
+
+public class Recognizer extends PointerType implements AutoCloseable {
+    public Recognizer(Model model) {

```

Comment:
We need to support different sample rates here

Reply:
The program seems to only support 16000, so it made sense to just hard-code it here instead of having the programmer hard-code it as you do in your other sample programs, and that way, the programmer won't think that he can just enter any number, which I did, thinking that it would work

Reply:
No, it can handle any sample rate above the model sample rate and the sample rate must be specified. 

It is critical for developer to think about sample rate, a mismatch is a source of serious accuracy issues and complains. Over years we had enormous amount of accuracy complains due to sample rate mismatch.

Reply:
I was not able to get it to work at all with anything other than 16000.  See https://github.com/alphacep/vosk-api/issues/424.  And in fact, the documentation also says that file must be 16Hz https://alphacephei.com/vosk/install.  Why do I get the error about specifying `allow_{upsample,downsample}` ?